src/main/java/org/broadinstitute/consent/http/service/EmailNotifierService.java

```diff
@@ -205,11 +199,15 @@ public void sendDisabledDatasetsMessage(User user, List<String> disabledDatasets
     public void sendClosedDataSetElectionsMessage(List<Election> elections) throws MessagingException, IOException, TemplateException {
         if(isServiceActive){
             Map<String, List<Election>> reviewedDatasets = new HashMap<>();
+            List<String> referenceIds = elections.stream().map(Election::getReferenceId).collect(Collectors.toList());
+            List<DarCollection> darCollections = collectionDAO.findDARCollectionsByReferenceIds(referenceIds);
```

Comment:
I know this was my suggestion, but it's safer to check if you have an empty list before passing it into a findBy method. Something like this:
```suggestion
            List<DarCollection> darCollections = referenceIds.isEmpty() ? List.of() : collectionDAO.findDARCollectionsByReferenceIds(referenceIds);
```